runtime/menu.vim

```diff
@@ -80,6 +80,9 @@ if has("gui_macvim")
   an <silent> 9998.350 Window.Select\ Previous\ Tab :tabprevious<CR>
   an 9998.360 Window.-SEP2-			    <Nop>
   an <silent> 9998.370 Window.Bring\ All\ To\ Front <Nop>
+  an <silent> 9998.380 Window.Stay\ On\ Bottom <Nop>
```

Comment:
looks good to me, but do you know any other macOS apps that has the same menu items? I think usually "Bring All To Front" item is the last item in `Window` menu (except window list).

    Window.-SEP2-
    Window.Stay\ On\ Top
    Window.Stay\ On\ Bottom
    Window.Stay\ Level\ Normal
    Window.-SEP3-
    Window.Bring\ All\ To\ Front